File: berkeleytime/submissions/accepted/berkeley_time_solution.py
def solve(N: int) -> str:
    if N == 0:
        return 'haha good one'
    elif N >= 180:
        return 'canceled'
    else:
        return 'berkeley' * (N // 10)


def main():
    T = int(input())
    for _ in range(T):
        N = int(input())
        print(solve(N))


if __name__ == '__main__':
    main()

File: berkeleytime/submissions/accepted/berkeley_time_solution.java
import java.io.*;

class Solution {
    static String solve(int N) {
        if (N == 0) {
            return "haha good one";
        } else if (N >= 180) {
            return "canceled";
        } else {
            String berkeleys = "";
            for (int i = 0; i < N; i += 10) {
                berkeleys += "berkeley";
            }
            return berkeleys + "time";
        }
    }
    
    static BufferedReader in = new BufferedReader(new InputStreamReader(System.in));
    static PrintWriter out = new PrintWriter(System.out);

    public static void main(String[] args) throws IOException {
        int T = Integer.parseInt(in.readLine());
        for (int i = 0; i < T; i++) {
            int N = Integer.parseInt(in.readLine());
            out.println(solve(N));
        }
        out.flush();
    }
}
